Makefile

```diff
@@ -0,0 +1,22 @@
+SHELL = '/bin/bash'
+export BUILD_TAG ?= 1.2.1335-r3.5.1-python3.7.1-conda-10
```

Comment:
I see we default to `latest` in the `docker-compose.yml`, don't know what's the best here but I wonder if use `latest` here as well for consistency is a good idea. (but maybe there was some reason to use this specific tag instead, sorry for the silly question)

Reply:
So if it isn't se it'll default to latest so it doesn't break. But if you're running docker-compose through Make then it'll set the environment variable. 

Once we start building and pushing images to ECR, I'd like to start auto tagging with GitHub Actions. So this is really to allow us to have some tagging flexibility going forward.